packages/babel-helper-create-class-features-plugin/src/decorators.ts

```diff
@@ -135,7 +129,7 @@ export function buildDecoratedClass(ref, path, elements, file) {
       .map(extractElementDescriptor.bind(file, node.id, superId)),
   );
 
-  let replacement = template.expression.ast`
+  let replacement: any = template.expression.ast`
```

Comment:
If we need to add an explicit type annotation for this, it might be worth checking if
```js
t.CallExpression & { arguments: [t.Expression, t.FunctionExpression] }
```
works.